ipatests/prci_definitions/nightly_ipa-4-6.yaml

```diff
@@ -59,7 +59,7 @@ jobs:
         timeout: 3600
         topology: *master_1repl
 
-  fedora-27/external_ca_1:
+  fedora-27/test_external_ca_TestExternalCA:
```

Comment:
I would rather keep the name external_ca_1 to be consistent between the branches.